local_deploy/deploy_ds.py

```diff
@@ -13,28 +13,13 @@
 from typing import List
 import requests
 from faiss_instant.encode_and_index import run as encode_and_index
-from square_auth.client_credentials import ClientCredentials
 import docker
 from docker.models.containers import Container
 from beir.util import download_and_unzip
+from utils import SharedVariables
 import tqdm
 
 
-def build_token() -> str:
-    os.environ["SQUARE_PRIVATE_KEY_FILE"] = os.path.join(os.getcwd(), "private_key.pem")
-    client_credentials = ClientCredentials(
-        keycloak_base_url="",
-        buffer=60,
-    )
-    return client_credentials()
-
-
-class SharedVariables:
-
-    token: str = build_token()
-    datastore_url: str = "http://localhost:7000"
-
-
```

Comment:
I think these two are not required.
`build_token` can just be replaced by:
```python
client_credentials = ClientCredentials()
```
Then, just call `client_credentials()` wherever you need a token. This is preferred, because this is also how we implement it everywhere else in square.
`datastore_url ` can just be a global variable at the beginning of each script that requires it. If you want to make it nicer, it can be loaded from an env var or given as an arugment.

Reply:
OK. What about now [d38b241](https://github.com/UKP-SQuARE/square-core/pull/354/commits/d38b2413e2d1dad9fe41aebca483f98bd3730f19)?